### PR TITLE
Bump outdated dependencies

### PR DESCRIPTION
**Pull request by ismail-syed**
* Updated outdated dependencies

Before:
screen shot 2018-08-30 at 1 40 03 pm

After: 
screen shot 2018-08-30 at 1 39 50 pm

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # Changelog
 
-## [Unreleased]
+<!-- ## [Unreleased] -->
+
+## [2.0.0] - 2018-08-30
 
 Breaking change: updated dependencies to use PostCSS 6.0.
```
Comment (kaelig) [inline CHANGELOG.md]: 6.0 -> 7

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+- Updated autoprefixer: `^6.7.6` -> `9.1.3` ([changelog](https://github.com/postcss/autoprefixer/blob/master/CHANGELOG.md))
+- Updated postcss: `^5.2.15` -> `7.0.2` ([changelog](https://github.com/postcss/postcss/blob/master/CHANGELOG.md))
 - Updated postcss-calc": `^5.3.1` -> `^6.0.1` ([changelog](https://github.com/postcss/postcss-calc/blob/master/CHANGELOG.md))
-- Updated postcss-flexbugs-fixes: `^2.1.0` -> `^3.2.0` ([changelog](https://github.com/luisrudge/postcss-flexbugs-fixes/blob/master/CHANGELOG.md))
+- Updated postcss-flexbugs-fixes: `^2.1.0` -> `4.1.0` ([changelog](https://github.com/luisrudge/postcss-flexbugs-fixes/blob/master/CHANGELOG.md))
+- Updated postcss-discard-comments: `^2.0.4` -> `4.0.0` ([changelog](https://github.com/cssnano/cssnano/blob/master/packages/postcss-discard-comments/CHANGELOG.md))
 - Updated postcss-selector-matches: `^2.0.5` -> `^3.0.1` ([changelog](https://github.com/postcss/postcss-selector-matches/blob/master/CHANGELOG.md))
 - Updated postcss-will-change: `^1.1.0` -> `^2.0.0` ([changelog](https://github.com/postcss/postcss-will-change/blob/master/CHANGELOG.md))
```
Comment (kaelig) [inline CHANGELOG.md:15]: If you happen to have time, they may need help on this https://github.com/postcss/postcss-will-change/issues/5

File: package.json [hunk 3]
```diff
@@ -23,15 +23,15 @@
...
-    "autoprefixer": "^7.2.5",
-    "postcss": "^6.0.16",
+    "autoprefixer": "9.1.3",
+    "postcss": "7.0.2",
     "postcss-calc": "^6.0.1",
-    "postcss-discard-comments": "^2.0.4",
-    "postcss-flexbugs-fixes": "^3.2.0",
+    "postcss-discard-comments": "4.0.0",
+    "postcss-flexbugs-fixes": "4.1.0",
     "postcss-selector-matches": "^3.0.1",
```
Comment (kaelig) [inline package.json]: Wanna pin those other dependencies? I guess `~` works too.

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,5 +20,6 @@ Breaking change: updated dependencies to use PostCSS 6.0.
 ## 1.0.0 - 2017-03-05
 - Initial release
 
-[Unreleased]: https://github.com/Shopify/postcss-shopify/compare/v1.0.1...master
+[Unreleased]: https://github.com/Shopify/postcss-shopify/compare/v2.0.0...master
+[2.0.0]: https://github.com/Shopify/postcss-shopify/compare/v1.0.1...2.0.0
```
Comment (kaelig) [inline CHANGELOG.md]: Missing `v` before `2.0.0` in `v1.0.1...2.0.0`

**Review by kaelig — APPROVED** [review]
Tested in polaris-styleguide, works great!

File: CHANGELOG.md (hunk 1 above)
Comment (ismail-syed) [reply CHANGELOG.md]: Good catch

File: package.json (hunk 3 above)
Comment (ismail-syed) [reply package.json]: pinned

**Pull request merged**